### PR TITLE
update for size mitigation of field P. issue #149

### PR DESCRIPTION
**Pull request by mpound**
override tclean params for field P. See issue #149

### CODE REVIEW

File: aces/pipeline_scripts/override_tclean_commands.json [hunk 1]
```diff
@@ -1768,7 +1800,7 @@
         "observation": "",
         "intent": "OBSERVE_TARGET#ON_SOURCE",
         "datacolumn": "corrected",
-        "imagename": "uid___A001_X15a0_Xfa.s38_0.Sgr_A_star_sci.spw33.cube.I.iter1",
+        "imagename": "uid___A001_X15a0_Xfa.Sgr_A_star_sci.spw33.cube.I.iter2",
```
Comment (keflavich) [inline aces/pipeline_scripts/override_tclean_commands.json:1803]: Why did you rename it here?

File: aces/pipeline_scripts/override_tclean_commands.json [hunk 2]
```diff
@@ -1822,18 +1852,20 @@
...
           ""
         ],
-        "niter": 30000000,
-        "gain": 0.1,
+        "usemask": "pb",
+        "pbmask": 0.5,
+        "niter": 50000000,
+        "cyclefactor": 3,
+        "nchan": 3836,
+        "width": "4882495Hz",
```
Comment (keflavich) [inline aces/pipeline_scripts/override_tclean_commands.json:1860]: Are you sure this is right?  4.8 MHz?  The previous value was 0.48 MHz

**Review by keflavich — CHANGES_REQUESTED** [review]

File: aces/pipeline_scripts/override_tclean_commands.json (hunk 2 above)
Comment (mpound) [reply aces/pipeline_scripts/override_tclean_commands.json:1860]: thanks. should have been "488249.5Hz".   will fix in new PR

Comment (keflavich) [reply aces/pipeline_scripts/override_tclean_commands.json:1860]: OK, generally it's better to correct the open PR rather than open a fresh one, though.
